test/extended/etcd/vertical_scaling.go

```diff
@@ -186,4 +199,103 @@ var _ = g.Describe("[sig-etcd][Feature:EtcdVerticalScaling][Suite:openshift/etcd
+	// 7) Then scale-down is validated by confirming the member removal and changes in the cluster membership
+	g.It("is able to vertically scale up and down when CPMS is disabled [apigroup:machine.openshift.io]", func() {
+		if cpmsActive {
+			// step 0: disable the CPMS
+			framework.Logf("Disable the CPMS")
+			err := scalingtestinglibrary.DisableCPMS(ctx, g.GinkgoT(), cpmsClient)
+			err = errors.Wrap(err, "pre-test: failed to disable the CPMS")
+			o.Expect(err).ToNot(o.HaveOccurred())
+
+			// re-enable CPMS after the test
+			defer func() {
+				framework.Logf("Re-enable the CPMS")
+				err := scalingtestinglibrary.EnableCPMS(ctx, g.GinkgoT(), cpmsClient)
+				err = errors.Wrap(err, "post-test: failed to re-enable the CPMS")
+				o.Expect(err).ToNot(o.HaveOccurred())
+			}()
+		}
+
+		framework.Logf("CPMS is disabled. The test will delete an existing machine and manually create a new machine to validate scale-down doesn't happen before scale-up event")
+
+		// step 1: delete a running machine
+		deletedMachineName, err := scalingtestinglibrary.DeleteSingleMachine(ctx, g.GinkgoT(), machineClient)
+		o.Expect(err).ToNot(o.HaveOccurred())
+		framework.Logf("Deleted machine %q", deletedMachineName)
+
+		// step 2: Verify the voting member count remain at 3 after the
+		// deletion of a machine and before a new machine is added to ensure scale-down hasn't occurred.
+		framework.Logf("Ensuring the etcd membership remains at 3 voting members to confirm that scale-down hasn't occurred")
+		err = scalingtestinglibrary.WaitOnExpectedVotingMembersCount(ctx, g.GinkgoT(), etcdClientFactory, kubeClient, 3)
+		o.Expect(err).To(o.HaveOccurred(), "voting member count shouldn't change from 3, as scale-down isn't expected to happen")
+
+		// step 3: add a new master node and wait until it is in Running state.
+		// The new machine should belong to the same index as the deleted machine
+		// to satisfy the requirement of at least 1 Ready Replica per index.
+		deletedMachineIndex, err := strconv.ParseInt(deletedMachineName[strings.LastIndex(deletedMachineName, "-")+1:], 10, 32)
+		newMachineName, err := scalingtestinglibrary.CreateNewMasterMachine(ctx, g.GinkgoT(), machineClient, deletedMachineIndex)
+		o.Expect(err).ToNot(o.HaveOccurred())
+		framework.Logf("Created machine %q", newMachineName)
+
+		err = scalingtestinglibrary.EnsureMasterMachine(ctx, g.GinkgoT(), newMachineName, machineClient)
+		err = errors.Wrapf(err, "scale-up: timed out waiting for machine (%s) to become Running", newMachineName)
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		// step 4: wait for 4 voting members to indicate scale-up event
+		framework.Logf("Waiting for etcd membership to show 4 voting members")
+		err = scalingtestinglibrary.EnsureVotingMembersCount(ctx, g.GinkgoT(), etcdClientFactory, kubeClient, 4)
+		err = errors.Wrap(err, "scale-up: timed out waiting for 4 voting members in the etcd cluster and etcd-endpoints configmap")
+		o.Expect(err).ToNot(o.HaveOccurred())
```

Comment:
I think this may not always succeed and will still be subject to the race outlined here:
https://github.com/openshift/origin/blob/441b43113c912b46b9cafaefc6e9078d651987dc/test/extended/etcd/vertical_scaling.go#L92-L96

So we can't realistically check for this so we can remove this.

It's enough to validate that scale-down doesn't happen (for healthy members) until we have a surplus of machines like we do in step 2.